Mogakco/Sources/Presentation/Chat/ViewController/ChatViewController.swift

```diff
@@ -58,27 +95,122 @@ final class ChatViewController: UICollectionViewController, ChatSidebarViewDeleg
         return true
     }
     
+    override func layout() {
+        configure()
+        layoutCollectionView()
+        layoutSideBar()
+        layoutBlackScreen()
+    }
+    
     private func configure() {
         configureSideBar()
         configureBlackScreen()
         configureNavigationBar()
     }
     
-    private func layout() {
-        layoutCollectionView()
-        layoutSideBar()
-        layoutBlackScreen()
+    override func bind() {
+        let input = ChatViewModel.Input(
+            backButtonDidTap: backButton.rx.tap.asObservable(),
+            studyInfoButtonDidTap: studyInfoButton.rx.tap.asObservable(),
+            selectedSidebar: sidebarView.tableView.rx.itemSelected.asObservable(),
+            sendButtonDidTap: messageInputView.sendButton.rx.tap.asObservable(),
+            inputViewText: messageInputView.messageInputTextView.rx.text.asObservable()
```

Comment:
```suggestion
            inputViewText: messageInputView.messageInputTextView.rx.text.orEmpty.asObservable()
```
orEmpty를 사용하면 viewModeld에서 optional로 받지 않아도 됩니다!

Reply:
반영했습니다! 감사합니다!